src/views/partials/settings.js

```diff
@@ -1,5 +1,183 @@
-export const settings = data => `
-<section>
-  <h1>This is the settings partial</h1>
-</section>
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import AppConstants from '../../app-constants.js'
+import { getMessage } from '../../utils/fluent.js'
+
+const emailNeedsVerificationSub = email => `
+  <span class='verification-required'>
+    ${getMessage('settings-email-verification-callout')}
+  </span>
+
+  <a class='js-settings-resend-email' data-email-id='${email.id}' href='#'>
+    ${getMessage('settings-resend-email-verification-link')}
+  </a>
 `
+
+const deleteButton = email => `
+  <button
+    data-subscriber-id='${email.subscriber_id}'
+    data-email-id='${email.id}'
+    class='settings-email-remove-button js-remove-email-button'
+  >
+    <img src='/images/icon-delete.svg'>
+  </button>
+`
+
+const createEmailItem = (email, breachCounts) => `
+  <li class='settings-email-item'>
+    <strong>
+      ${email.primary
+          ? `${getMessage('settings-email-label-primary', { email: email.email })}`
+          : email.email}
+    </strong>
+    ${email.verified
+        ? getMessage('settings-email-number-of-breaches-info', {
+            breachCount: breachCounts.get(email.email)
+          })
+        : emailNeedsVerificationSub(email)}
+
+    ${email.primary ? '' : deleteButton(email)}
+  </li>
+`
+
+// Moves the primary email to the front and sorts the rest alphabeticaly.
+const getSortedEmails = emails => [...emails].sort((a, b) => {
+  if (a.primary) {
+    return -1
+  }
+
+  if (b.primary) {
+    return 1
+  }
+
+  return a.email.localeCompare(b.email)
+})
+
+const createEmailList = (emails, breachCounts) => `
+  <ul class='settings-email-list'>
+    ${getSortedEmails(emails)
+      .map(email => createEmailItem(email, breachCounts))
+      .join('')}
+  </ul>
+`
+
+const optionInput = (csrfToken, { isChecked, option }) => `
+  <input
+    checked='${isChecked}'
```

Comment:
Be careful here, I believe `checked` == `checked="true"` == `checked="false"`.  So, having `checked="false"` is actually specifying the checkbox _is_ checked.

You probably need to use a conditional here:
```suggestion
    ${isChecked ? 'checked' : ''}
```

https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes